CHANGELOG.md

```diff
@@ -7,13 +7,24 @@ This file was introduced starting Kani 0.23.0, so it only contains changes from
 ## [0.34.0]
 
 ### Breaking Changes
-* Change default solver to CaDiCaL by @celinval in https://github.com/model-checking/kani/pull/2557
+* Change default solver to CaDiCaL ([pull request](https://github.com/model-checking/kani/pull/2557) by @celinval)
 By default, Kani will now run CBMC with CaDiCaL, since this solver has outperformed Minisat in most of our benchmarks.
 User's should still be able to select Minisat (or a different solver) either by using `#[solver]` harness attribute,
 or by passing `--solver=<SOLVER>` command line option.
 
 ## What's Changed
 
+* Allow specifying the scheduling strategy in #[kani_proof] for async functions ([pull request](https://github.com/model-checking/kani/pull/1661) by @fzaiser)
+* Support for stubbing out foreign functions ([pull request](https://github.com/model-checking/kani/pull/2658) by @feliperodri)
+* Coverage reporting without a need for cbmc-viewer ([pull request](https://github.com/model-checking/kani/pull/2609) by @adpaco-aws)
+* Add support to array-based SIMD ([pull request](https://github.com/model-checking/kani/pull/2633) by @celinval)
+* Add unchecked/SIMD bitshift checks and disable CBMC flag ([pull request](https://github.com/model-checking/kani/pull/2630) by @reisnera)
+* Bump CBMC to v5.89.0 ([pull request](https://github.com/model-checking/kani/pull/2662) by @remi-delmas-3000)
+* Update Rust toolchain to nightly 2023-08-04 ([pull request](https://github.com/model-checking/kani/pull/2661) by @remi-delmas-3000)
```

Comment:
I'd rather use the default format to conform with the format in release notes for other versions.

Reply:
Also, isn't this missing #2663 ?

Reply:
Reverted the beautification commit, and added #2663 back in (I wasn't sure whether that was to be considered user-facing, but the spurious verification failures that it addresses surely are).